src/auth.utils.ts

```diff
@@ -22,4 +21,26 @@ if (process.env.AUTH_PROVIDER == "local") {
   });
 }
 
-export { getSession };
+export async function getSoknadOnBehalfOfToken(session: SessionWithOboProvider) {
```

Comment:
Denne funksjon gir oss muligheten til å hente lokal token fra `.env.development` filen for å kunne snakke til DEV miljøet.